proposals/csharp-12.0/collection-expressions.md

```diff
@@ -228,6 +235,8 @@ If the target type is a *struct* or *class type* that implements `System.Collect
 * For each element in order:
   * If the element is an *expression element*, the applicable `Add` instance or extension method is invoked with the element *expression* as the argument. (Unlike classic [*collection initializer behavior*](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#117154-collection-initializers), element evaluation and `Add` calls are not necessarily interleaved.)
   * If the element is a *spread element* then one of the following is used:
+    * If the spread element expression is a *collection expression*, then the collection expression elements are evaluated in order as if the elements were in the containing collection expression.
+    * If the spread element expression is a *conditional expression*, then the condition is evaluated and if `true` the second-, otherwise the third-operand is evaluated as if the operand was an element in the containing collection expression.
```

Comment:
Only curious: Is there any where in the spec that restates this for purposes like target-typed new _inside_ a conditional expression looking _outward_? Could both directions be unified somehow in some definition of type inference for any language constructs that support "seeing through" them?

Reply:
Probably should have commented this in the convertibility section.